src/metabase/api/pulse.clj

```diff
@@ -275,6 +276,18 @@
                           render/*include-buttons* true]
                   (render/render-pulse-card-for-display (metabase.pulse/defaulted-timezone card) card result))]])}))
 
```

Comment:
My only big question here is if we want to enable this publicly, because then it's a supported API. I'd much rather expose it only if the dev profile is active. I don't see why a customer would want this, but it's exceptionally valuable for development. WDYT?